modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -459,4 +459,89 @@ definition {
 		}
 	}
 
+	@description = "LPS-173894 - Verify that after updating a custom object entry, triggered user notifications using Object Author and Current User terms are filled correctly for the To and Subject fields"
+	@priority = 5
```

Comment:
You need to add the property of the feature flag and the acceptance property too.
_property portal.acceptance = "true";_
_property custom.properties = "feature.flag.LPS-171625=true";_